packages/ketcher-react/src/script/ui/component/actionmenu.jsx

```diff
@@ -60,14 +60,22 @@ export function showMenuOrButton(action, item, status, props) {
   return item.component(props)
 }
 
-function ActionButton({ name, action, status = {}, onAction }) {
+function ActionButton({
+  name,
+  action,
+  status = {},
+  onAction,
+  disableableButtons,
+  indigoVerification
+}) {
   // eslint-disable-line no-shadow
   const shortcut = action.shortcut && shortcutStr(action.shortcut)
   const menuRef = useRef(null)
+  const disabled = indigoVerification && disableableButtons.includes(name)
   return (
     <button
       ref={menuRef}
-      disabled={status.disabled}
+      disabled={status.disabled || disabled}
```

Comment:
should be better to move `status.disabled || ` to `disable` variable assign. because in case when `status.disabled == true` you will perform unnecessary search in array

Reply:
40beb7d739bbbc9d91c04547f0a043db56fb554d